gruntfile.js

```diff
@@ -9,167 +9,55 @@ module.exports = function (grunt) {
-    'concat',
-    'clean:tmpl',
-    'cssmin',
-    'replace',
-    'uglify',
-    'copy:assets',
-    'filerev',
-    'usemin',
-    'clean:tmp'
-  ]);
-  grunt.registerTask('release-linux-386', [
-    'config:prod',
-    'clean:all',
-    'if:linux386BinaryNotExist',
-    'html2js',
-    'useminPrepare:release',
-    'recess:build',
-    'concat',
-    'clean:tmpl',
-    'cssmin',
-    'replace',
-    'uglify',
+    'before-copy',
     'copy:assets',
-    'filerev',
-    'usemin',
-    'clean:tmp'
+    'after-copy'
   ]);
-  grunt.registerTask('release-linux-amd64', [
-    'config:prod',
-    'clean:all',
-    'if:linuxAmd64BinaryNotExist',
-    'html2js',
-    'useminPrepare:release',
-    'recess:build',
-    'concat',
-    'clean:tmpl',
-    'cssmin',
-    'replace',
-    'uglify',
-    'copy:assets',
-    'filerev',
-    'usemin',
-    'clean:tmp'
-  ]);
-  grunt.registerTask('release-linux-arm', [
-    'config:prod',
-    'clean:all',
-    'if:linuxArmBinaryNotExist',
-    'html2js',
-    'useminPrepare:release',
-    'recess:build',
-    'concat',
-    'clean:tmpl',
-    'cssmin',
-    'replace',
-    'uglify',
-    'copy',
-    'filerev',
-    'usemin',
-    'clean:tmp'
-  ]);
-  grunt.registerTask('release-linux-arm64', [
-    'config:prod',
-    'clean:all',
-    'if:linuxArm64BinaryNotExist',
-    'html2js',
-    'useminPrepare:release',
-    'recess:build',
-    'concat',
-    'clean:tmpl',
-    'cssmin',
-    'replace',
-    'uglify',
-    'copy',
-    'filerev',
-    'usemin',
-    'clean:tmp'
-  ]);
-  grunt.registerTask('release-linux-ppc64le', [
-    'config:prod',
-    'clean:all',
-    'if:linuxPpc64leBinaryNotExist',
+  grunt.registerTask('build', [
+    'config:dev',
+    'clean:app',
+    'shell:buildBinary:linux:amd64',
     'html2js',
-    'useminPrepare:release',
+    'useminPrepare:dev',
     'recess:build',
     'concat',
     'clean:tmpl',
-    'cssmin',
     'replace',
-    'uglify',
     'copy',
-    'filerev',
-    'usemin',
-    'clean:tmp'
+    'after-copy'
   ]);
-  grunt.registerTask('release-windows-amd64', [
-    'config:prod',
-    'clean:all',
-    'if:windowsAmd64BinaryNotExist',
+  grunt.registerTask('before-copy', [
     'html2js',
     'useminPrepare:release',
     'recess:build',
     'concat',
     'clean:tmpl',
     'cssmin',
     'replace',
-    'uglify',
-    'copy',
-    'filerev',
-    'usemin',
-    'clean:tmp'
+    'uglify'
   ]);
-  grunt.registerTask('release-darwin-amd64', [
-    'config:prod',
-    'clean:all',
-    'if:darwinAmd64BinaryNotExist',
-    'html2js',
-    'useminPrepare:release',
-    'recess:build',
-    'concat',
-    'clean:tmpl',
-    'cssmin',
-    'replace',
-    'uglify',
-    'copy',
+  grunt.registerTask('after-copy', [
     'filerev',
     'usemin',
-    'clean:tmp'
+    'clean:tmp' 
   ]);
+  grunt.task.registerTask('release', 'release:<platform>:<arch>', function(p, a) {
+	var cp = ( p === 'linux' && ( a === '386' || a === 'amd64' ) ) ? 'copy:assets' : 'copy';
```

Comment:
I got lost with the reference to `copy:bundle`, because I don't see when is it called.

> Every release task should only use the copy:assets task.

I replaced `var cp = ( p === 'linux' && ( a === '386' || a === 'amd64' ) ) ? 'copy:assets' : 'copy';` with `copy:assets` for every release.

> Note: as an enhancement minified vendor and angular libraries should only be used during release, unminified libraries should be used during development.

That's the actual behaviour, isn't it?